platform/ios/src/MGLMapView.mm

```diff
@@ -2150,30 +2156,45 @@ - (void)handleTwoFingerDragGesture:(UIPanGestureRecognizer *)twoFingerDrag
             twoFingerDrag.state = UIGestureRecognizerStateEnded;
             return;
         }
-
-        CGFloat gestureDistance = CGPoint([twoFingerDrag translationInView:twoFingerDrag.view]).y;
-        CGFloat slowdown = 2.0;
-
-        CGFloat pitchNew = initialPitch - (gestureDistance / slowdown);
-
-        CGPoint centerPoint = [self anchorPointForGesture:twoFingerDrag];
-
-        MGLMapCamera *oldCamera = self.camera;
-        MGLMapCamera *toCamera = [self cameraByTiltingToPitch:pitchNew];
-
-        if ([self _shouldChangeFromCamera:oldCamera toCamera:toCamera])
-        {
-            self.mbglMap.jumpTo(mbgl::CameraOptions()
+        
+        CGPoint west = [twoFingerDrag locationOfTouch:0 inView:twoFingerDrag.view];
+        CGPoint east = [twoFingerDrag locationOfTouch:1 inView:twoFingerDrag.view];
```

Comment:
Calling these `east` and `west` might be somewhat misleading, since a map could be rotated in such a way that these are geographically incorrect. A common naming scheme for when there’s directional uncertainty is `leading` and `trailing` (or really, in this case `left` and `right` would also be appropriate, I think).

In any case, let’s append something like `TouchPoint` to make these more descriptive.